<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFilterDialog</class>
 <widget class="QDialog" name="EditFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit filter</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>382</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>12</height>
      </size>
     </property>
     <property name="text">
      <string>Enter a custom name for the filter (leave blank for default).</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="customNameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>382</width>
       <height>12</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>12</height>
      </size>
     </property>
     <property name="text">
      <string>Apply a convolution matrix.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="matrixGridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <item row="3" column="0">
      <widget class="QSpinBox" name="spinBox_4">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSpinBox" name="spinBox_5">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="spinBox_2">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="spinBox_3">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QSpinBox" name="spinBox">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="spinBox_19">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QSpinBox" name="spinBox_20">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_7">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinBox_16">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_6">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_9">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_10">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_8">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_11">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_12">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinBox_14">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_13">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinBox_15">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinBox_17">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinBox_18">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="spinBox_21">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="spinBox_22">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QSpinBox" name="spinBox_23">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QSpinBox" name="spinBox_24">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QSpinBox" name="spinBox_25">
       <property name="whatsThis">
        <string>A coefficient value which will be used to compute the output map.</string>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>customNameLineEdit</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>spinBox_6</tabstop>
  <tabstop>spinBox_7</tabstop>
  <tabstop>spinBox_8</tabstop>
  <tabstop>spinBox_9</tabstop>
  <tabstop>spinBox_10</tabstop>
  <tabstop>spinBox_11</tabstop>
  <tabstop>spinBox_12</tabstop>
  <tabstop>spinBox_13</tabstop>
  <tabstop>spinBox_14</tabstop>
  <tabstop>spinBox_15</tabstop>
  <tabstop>spinBox_16</tabstop>
  <tabstop>spinBox_17</tabstop>
  <tabstop>spinBox_18</tabstop>
  <tabstop>spinBox_19</tabstop>
  <tabstop>spinBox_20</tabstop>
  <tabstop>spinBox_21</tabstop>
  <tabstop>spinBox_22</tabstop>
  <tabstop>spinBox_23</tabstop>
  <tabstop>spinBox_24</tabstop>
  <tabstop>spinBox_25</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>EditFilterDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>EditFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
